modules/collect/src/main/java/com/opengamma/strata/collect/result/FailureItem.java

```diff
@@ -132,13 +132,12 @@ static FailureItem of(FailureReason reason, String message, int skipFrames) {
    * @param reason  the reason
    * @param message  the failure message, not empty
    * @param attributes the attributes associated with this failure
-   * @param skipFrames  the number of caller frames to skip, not including this one
    * @return the failure
    */
-  public static FailureItem of(FailureReason reason, String message, Map<String, String> attributes, int skipFrames) {
+  public static FailureItem of(FailureReason reason, String message, Map<String, String> attributes) {
```

Comment:
I've had second thoughts about including this method. I think it might be confusing. What if the user includes named placeholders in the message? Will they expect the attribute values to be substituted? Also, for symmetry you might expect the attribute map to be added to other factory methods. Better to keep things simple and get rid of it seeing as we said we didn't have a use case for it now. We can always add it later if we need to.

P.S. Sorry for changing my mind.

Reply:
Done